docs/userGuide/syntax/lists.md

```diff
@@ -42,6 +42,22 @@ You can use any of the [icons](../formattingContents.html#icons) supported by Ma
 If you customize any item on a certain level, you must also customize the first item on that level. If not, the list will revert to its uncustomized form.
 </box>
 
+**You can add an icon that is only for a specific item by using the `i-one-off` attribute.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* Item 1 { icon="glyphicon-education" }
+* Item 2 { icon="fas-code-branch" i-one-off=true }
+  * Item 2.1
+  * Item 2.2
+* Item 3
+  </variable>
+</include>
+
+Setting `i-one-off=true` is preferred but `i-one-off="true"` works as well.
```

Comment:
Move this line below and put it in a tip box

Reply:
tbh I think we don't even need this. We shouldn't be encouraging this behaviour in the first place to use a string over boolean.